ci-operator/step-registry/hypershift/openstack/destroy/hypershift-openstack-destroy-chain.yaml

```diff
@@ -0,0 +1,22 @@
+chain:
+  as: hypershift-openstack-destroy
+  steps:
+  - as: destroy
+    cli: latest
+    commands: |-
+      set -exuo pipefail
```

Comment:
I usually avoid `-x` in the CI because it invariably ends up with leaked credentials. Is this a leftover, or are you that brave?

Reply:
done